src/presentation/http/router/note.test.ts

```diff
@@ -385,6 +385,65 @@ describe('Note API', () => {
     test.todo('Returns 400 when parentId has incorrect characters and lenght');
   });
 
+  describe('DELETE /note/:notePublicId', () => {
+    test('Should remove all note relations contains note id', async () => {
+      const accessToken = global.auth(2);
+      const correctID = 'f43NU75weU';
+
+      let response = await global.api?.fakeRequest({
+        method: 'DELETE',
+        headers: {
+          authorization: `Bearer ${accessToken}`,
+        },
+        url: `/note/${correctID}`,
+      });
+
+      expect(response?.statusCode).toBe(200);
+
+      /**
+       * Checks if the note itself has been deleted
+       */
+      response = await global.api?.fakeRequest({
+        method: 'GET',
+        headers: {
+          authorization: `Bearer ${accessToken}`,
+        },
+        url: `/note/${correctID}`,
+      });
+
+      expect(response?.json()).toStrictEqual({ message: 'Note not found' });
+
+      const childNote = 'Uyd8TgkdA0';
+
+      /**
+       * Checks if the note saved as a parent has been deleted
+       */
+      response = await global.api?.fakeRequest({
+        method: 'GET',
+        headers: {
+          authorization: `Bearer ${accessToken}`,
+        },
+        url: `/note/${childNote}`,
+      });
+
+      const expectedResponse = {
+        'note': {
+          'id': 56,
+          'publicId': 'Uyd8TgkdA0',
+          'creatorId': 2,
+          'content': null,
+          'createdAt': '2023-10-16T13:49:19.000Z',
+          'updatedAt': '2023-10-16T13:49:19.000Z',
+        },
+        'accessRights': {
+          'canEdit': true,
+        },
+      };
```

Comment:
do not use expectedResponse

Reply:
just dont use variable for data `expect(response?.json()).toStrictEqual(your data)`